src/main/java/pb360/data/entity/RestApi.java

```diff
@@ -0,0 +1,85 @@
+package pb360.data.entity;
+
+import java.util.Date;
+import java.util.List;
+
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import pb360.model.MessageObject;
+
+@Document(collection= "RestApi")
+
+public final class RestApi {
+	private @JsonProperty("restId") String restId;
```

Comment:
why u added JsonProperties on an entry of mongoDb?

Reply:
So, JsonProperties  should only be on Model right ?